src/arrayOfPeople.js

```diff
@@ -37,8 +37,13 @@
  *
  * @return {number[]}
  */
+
 function arrayOfPeople(people) {
-  // write code here
+  let arr = [];
+  people.map((el) => {
```

Comment:
Use `map` only in case you need a new array with processed values. Here is no use case for it. Replace `map` with a more efficient loop.